site/layouts/partials/page-meta-links.html

```diff
@@ -10,7 +10,26 @@
 {{ if $gh_repo }}
 <div class="td-page-meta ml-2 pb-1 pt-2 mb-0">
 {{ $editURL := printf "%s/edit/main/site/content/%s/%s" $gh_repo ($.Site.Language.Lang) $path }}
-{{ $issuesURL := printf "%s/issues/new?title=%s" $gh_repo (htmlEscape $.Title )}}
+
+{{ $issuesBodyTemplate := `
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+`}}
```

Comment:
Can we rephrase these pre-filled sentences so they sounds more like "documentation issue" rather than "feature request"? I feel the existing bug_report template is more suitable as a reference here but could also use some revise. @roberthbailey @markmandel any suggestions to the wording here?

Reply:
Oooh! nifty to see this get fixed!

This was the template I put together - lemme know what you think:

```
**Link to page**: 

$pageURL

**Description of documentation issue**:
A clear and concise description of what the problem is with this documentation - e.g. "I find it hard to understand...."

**What improvements to documentation would you like to see?**:
A clear and concise description of what modifications or additions to documentation you would like to see.
```

I'm not quite sure how to get `$pageURL` in the middle of the template like that though.

 I was also wondering if the bug title should have something more to it than just the page title - but I couldn't come up with anything better.

Reply:
> I'm not quite sure how to get $pageURL in the middle of the template like that though.
string formatting(%s) helped to use the variable $pageURL.

> I was also wondering if the bug title should have something more to it than just the page title - but I couldn't come up with anything better.
Sorry, I am unsure about this. Could you please provide an example for `something more` for the bug title?

Reply:
> Sorry, I am unsure about this. Could you please provide an example for `something more` for the bug title?

Yeah, i couldn't think of anything better 😄 that was me just saying "does anyone have any good ideas" ?

Oooh, that made me realise, we should add the [area/site](https://github.com/googleforgames/agones/labels/area%2Fsite) tag as well please!

With both tags, we probably don't need anything extra in the tittle - the label metadata will be sufficient.

Reply:
The `area/site` label has included to the issues👍🏻